server/migration/1607616094276-ContractorAffiliationUpdate.ts

```diff
@@ -0,0 +1,13 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
```

Comment:
The migration generator doesn't format according to our eslint rules unfortunately, so these have to be formatted manually. Can you just format this file to match the eslint rules?